packages/core/src/Page/PageVueServerRenderer.ts

```diff
@@ -78,8 +89,8 @@ async function renderVuePage(compiledVuePage) {
  *
  * Credits to Dominic
  */
-function requireFromString(src, filename) {
-  const m = new module.constructor();
+function requireFromString(src : string, filename : string) {
+  const m = new (module.constructor as any)();
```

Comment:
If I don't do this, typescript will complain: `This expression is not constructable.   Type 'Function' has no construct signatures.`